javascript/core/flow-control/loops-part-1.md

```diff
@@ -82,16 +71,20 @@ while (i < 10) {
   console.log(i);
   i++;
 }
+
+// 0, 1, 2, 3, 4, 5, 6, 7, 8, 9
 ```
 
 **do while**
 
-*do while* loops are similar to while loops but have the important difference that they will always run at least once before the condition is checked:
+`do`/`while` loops are similar to `while` loops but have the important difference that they will always run at least once before the condition is checked:
```

Comment:
I feel like this would look better and keep consistency. what do you think?
```suggestion
`do...while` loops are similar to `while` loops but have the important difference that they will always run at least once before the condition is checked:
```

Reply:
Agreed, except it should be `..` instead of `...` right?

Reply:
yes, it should :+1:  will make the changes